backend/src/main/java/gov/cdc/usds/simplereport/api/apiuser/UserAccountCreationController.java

```diff
@@ -231,13 +229,12 @@ public FactorAndQrCode getAuthQrCode(
    *     their security key.
    */
   @PostMapping("/enroll-security-key-mfa")
-  public JsonNode enrollSecurityKeyMfa(@SessionAttribute String userId, HttpServletRequest request)
+  public FactorAndActivation enrollSecurityKeyMfa(
+      @SessionAttribute String userId, HttpServletRequest request)
       throws OktaAuthenticationFailureException {
-    JSONObject enrollResponse = _oktaAuth.enrollSecurityKey(userId);
-    request.getSession().setAttribute(FACTOR_ID_KEY, enrollResponse.getString(FACTOR_ID_KEY));
-    return JsonNodeFactory.instance
-        .objectNode()
-        .put("activation", enrollResponse.getJSONObject("activation").toString());
+    var enrollResponse = _oktaAuth.enrollSecurityKey(userId);
```

Comment:
honestly, I personally don't like it, I constantly find myself trying to infer what the type is. Takes away from readability and adds nothing in return.

Reply:
I would make the argument that it actively hurts you while writing code, when you write down the type first, your IDE can look at the type and suggest a good name for the variable, I find myself taking the suggested name like 90% of the time. 
When you use `var`, the IDE has no context, so it doesn't give you good suggestions since it can be anything!

Reply:
I personally think it's usage is fine especially when the returned value or variable name is clear, such as `var updateRequest = new UpdateUserRequest();` or `var userList = userApi.getUsers(...);`. It's not so good if the type is not clear or bad variable names are used. 

I think the IDE should handle most things with auto-naming pretty well. 
<img width="384" alt="image" src="https://github.com/CDCgov/prime-simplereport/assets/10108172/0c9adbde-d26e-4248-ac25-fa8a716166d1">
<img width="362" alt="image" src="https://github.com/CDCgov/prime-simplereport/assets/10108172/b8988fd6-5dee-4ebf-b09a-456cc347fafd">
<img width="658" alt="image" src="https://github.com/CDCgov/prime-simplereport/assets/10108172/afa38c35-dcb7-4caf-8212-cec7605ffa9d">
vs 
<img width="523" alt="image" src="https://github.com/CDCgov/prime-simplereport/assets/10108172/1bc5bba3-94cd-4293-81d9-206815c31434">
<img width="410" alt="image" src="https://github.com/CDCgov/prime-simplereport/assets/10108172/ca19763c-8573-4d11-872a-35fb3ee056f0">

And then if you just hover over the variable it can explicitly tell you the type. 
<img width="460" alt="image" src="https://github.com/CDCgov/prime-simplereport/assets/10108172/1cd0af5a-1697-49bb-9a11-fd8c18c01754">

Read more about using `var`: https://openjdk.org/projects/amber/guides/lvti-style-guide